plugins/katello/developers.md

```diff
@@ -197,30 +197,26 @@ npm run lint:test # Run both linting and testing
 
 This guide covers how to peform a community release of Katello. The guide will reference the term 'X.Y' which reprsents the major and minor version that you are releasing. Throughout the guide we will use 2.0 and 2.1 as examples to represent the currently released community version (2.0) and the version that you are working on releasing (2.1).
 
-### Step 0: Code Name
+The Katello release process has two distinct roles that work together to get a release out.
 
-Pick a release code name based on a style of beer previously unused. Examples from previous releases are Oktoberfest and Winter Warmer.
+ * release owner: manages the release, ensuring that dates are being hit, communicated to the community, issues are being managed in Redmine
+ * release packager: handles creating source packages, testing pipelines, Koji setup and RPM building
```

Comment:
I think in Foreman we've used release engineer. Should release be capitalized here?

Reply:
If I capitalize one, I should capitalize both words. I can go either way. Preference?